src/averageAges.js

```diff
@@ -14,14 +14,21 @@
  *
  * @return {number}
  */
+
+const calculateAverages = (ages, count) => ages.reduce(
```

Comment:
Rename 'calculateAverages' function to 'calculateAverage' for a more appropriate and accurate naming.